manifests/dashboards.pp

```diff
@@ -0,0 +1,34 @@
+# @summary Configures Grafana dashboards.
+#
+# Configures Grafana dashboards.
+#
+# @api private
+class puppet_metrics_dashboard::dashboards {
+  $_overwrite_dashboards_ensure = $puppet_metrics_dashboard::overwrite_dashboards ? {
```

Comment:
Has the logic around adding the dashboard examples changed here?  Previously you'd need to set `add_dashboard_examples = true` and you'd get example dashboards depending on what influxdb database name you supplied.  I can't seem to get the example dashes with this change.

Reply:
no logic change. `add_dashboard_examples` controls the inclusion of this entire class.